rules/linux/discovery_esxi_software_via_find.toml

```diff
@@ -59,23 +59,22 @@ tags = [
 ]
 timestamp_override = "event.ingested"
 type = "eql"
-
 query = '''
-process where host.os.type == "linux" and event.type == "start" and event.action in ("exec", "exec_event", "executed", "process_started")
- and process.name == "find" and process.args : ("/etc/vmware/*", "/usr/lib/vmware/*", "/vmfs/*")
+process where host.os.type == "linux" and event.type == "start" and
+event.action in ("exec", "exec_event", "executed", "process_started") and process.name == "find" and
+process.args : ("/etc/vmware/*", "/usr/lib/vmware/*", "/vmfs/*") and 
+not process.parent.executable == "/usr/lib/vmware/viewagent/bin/uninstall_viewagent.sh"
```

Comment:
Is there an install script we need to ignore for vmware as well that may use the same find process?

Reply:
Looking at telemetry I think this should be enough. This rule in general is not noisy; it was just 1 FP.